pkg/operator/controller/machineconfig/controller.go

```diff
@@ -36,6 +37,7 @@ import (
 //+kubebuilder:rbac:groups=machineconfiguration.openshift.io,resources=machineconfigs,verbs=get;list;watch;create;delete
 //+kubebuilder:rbac:groups=machineconfiguration.openshift.io,resources=machineconfigpools,verbs=get;list;watch;create;delete
 //+kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch;patch
+//+kubebuilder:rbac:groups="",resources=events,verbs=create;patch
```

Comment:
For what do we need to patch events?

Reply:
We do not need permission for Patch on Events, will correct it.

Reply:
comment incorporated